package/knot-storage/Config.in

```diff
@@ -0,0 +1,10 @@
+config BR2_PACKAGE_KNOT_STORAGE
+    bool "knot-storage"
+    depends on BR2_PACKAGE_HOST_GO_ARCH_SUPPORTS
+    depends on BR2_PACKAGE_HOST_GO_CGO_LINKING_SUPPORTS
+    depends on BR2_PACKAGE_RABBITMQ_SERVER
+    depends on BR2_PACKAGE_MONGODB
+    depends on BR2_PACKAGE_KNOT_BABELTOWER
```

Comment:
If any of these dependencies are required to build the package, add it to the mk file.

Reply:
As we discussed, only libraries need to be added as dependencies in the `.mk` file.